cardano-node/src/Cardano/Node/Startup.hs

```diff
@@ -209,3 +209,25 @@ prepareNodeInfo ptcl (SomeConsensusProtocol whichP pForInfo) tc nodeStartTime =
     case tcNodeName tc of
       Just aName -> return aName
       Nothing    -> pack <$> getHostName
+
+-- | This information is taken from 'BasicInfoShelleyBased'. It is required for
+--   'cardano-tracer' service (particularly, for RTView).
+data NodeStartupInfo = NodeStartupInfo {
```

Comment:
If it is taken from `BasicInfoShelleyBased` why are we essentially duplicating the type?

Reply:
@Jimbo4350 

1. `NodeStartupInfo` takes not _all_ fields from `BasicInfoShelleyBased`.
2. `NodeStartupInfo` may pass another data in the future, not only from `BasicInfoShelleyBased`.